docs/ops/pooling/MaxPool_1.md

```diff
@@ -18,6 +18,14 @@
   * **Default value**: None
   * **Required**: *yes*
 
+* *dilations*
```

Comment:
For the MaxPool-8 we also want to add a new attribute defining the padding value. This change is needed to resolve issue that TF/ONNX and Caffe have different default value for the padded elements (TF/ONNX - 0, while Caffe - minus infimum). Some details in the ticket 35367

Reply:
Added `pads_value`

Reply:
@lazarevevgeny @postrational Could you please modify `pads_value` to be a string instead of float value to limit space of possible values? As I understand there is no reason for supporting anything except for `0` and `-inf`. Also float padding doesn't make any sense for pooling on integer data, but it will complicate implementation a lot to support such cases.

```
* *pads_value*

    * **Description**: *pads_value* will be used to fill pixels added during padding. If not specified, the new pixels will be filled in with minimum value for the type.
    * **Range of values**:
      * *MIN*
      * *ZERO*
    * **Type**: string
    * **Default value**: *MIN*
    * **Required**: *no*
```

Reply:
@lazarevevgeny Also could you please clarify the issue? As I understand current IE and oneDNN implementation is aligned with TF (pad: -inf):
```python
>>> import tensorflow as tf
>>> import numpy as np
>>> a = np.array([-0.8, -0.3, 0.4,
...               -0.1, -0.2, 0.3,
...               -1.0, -2.0, -3.0])
>>> a = np.reshape(a, (1, 3, 3, 1))
>>> x = tf.constant(a)
>>> y = tf.nn.max_pool2d(x, ksize=[3, 3], strides=[2, 2], padding='SAME')
>>> y
<tf.Tensor: shape=(1, 2, 2, 1), dtype=float64, numpy=
array([[[[-0.1],
         [ 0.4]],

        [[-0.1],
         [ 0.3]]]])>
```